test/e2e/scheduling/priorities.go

```diff
@@ -399,3 +443,27 @@ func addRandomTaitToNode(cs clientset.Interface, nodeName string) *v1.Taint {
 	framework.ExpectNodeHasTaint(cs, nodeName, &testTaint)
 	return &testTaint
 }
+
+func UpdateMemoryOfNode(c clientset.Interface, nodeName string, value int64) error {
+	node, err := c.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
+	framework.ExpectNoError(err)
+	oldData, err := json.Marshal(node)
+	if err != nil {
+		return err
+	}
+	node.Status.Allocatable[v1.ResourceMemory] = *resource.NewQuantity(value, resource.BinarySI)
+	newData, err := json.Marshal(node)
+	if err != nil {
+		return err
+	}
+	patchBytes, err := strategicpatch.CreateTwoWayMergePatch(oldData, newData, v1.Node{})
+	if err != nil {
+		return err
+	}
+
+	_, err = c.CoreV1().Nodes().Patch(string(node.Name), types.StrategicMergePatchType, patchBytes)
+	if err != nil {
```

Comment:
nit: This if statement is not needed. At this point it returns 'err' regardless.